### DISCUSSION TITLE
Dynamic render mode

### DISCUSSION THREAD

**Discussion by Thr0nSK** [maruohon/litematica#353] [Ideas]
It would be great if we could set a dynamic render mode wehere it would render all blocks below the player / above the player with the option to configure the distance from the player.

**Comment by maruohon** [comment]
There is an option for a "player-following" render layer mode at least in the 1.12.2 version. And I think I also ported an older, simpler version of that to 1.16.4 at some point. I don't remember if it's in the version that is on CurseForge though, if not, then it might be i nthe version on my server (linked from the snapshot versions section on the CF page). I think the simpler version's option to enable that feature is in the Generic config category, whereas the newer version in 1.12.2 has the options in the Render Layers menu directly.

**Reply by Thr0nSK** [reply]
I am using the latest 1.16.5 version and wasn't able to find anything like that there. Btw. when could we expect a 1.17 port? 😇 No pressure, just asking 😅 Thank you!

**Reply by maruohon** [reply]
Looks like it was ported on 11th of February, and the version on CurseForge in from January. So get the latest build from my server: https://masa.dy.fi/mcmods/client_mods/?mcver=1.16.4

There are also 1.17-pre4 versions on my server already since a few days, and I'll make a separate 1.17.0 build later today and upload those to CF as well.
